docs/TROUBLESHOOTING.md

```diff
@@ -48,6 +48,34 @@ Once the SSH bastion has been setup, you can use either method to access the Win
   ```
 * You can now RDP into the Windows node at *localhost:2020* using an RDP client
 
+## Rebooting a Windows node
+
+In general, the operator minimize disruptions and avoid node reboots whenever possible. Certain operations and
```

Comment:
in general, the operator tries to minimize distruptions?

Reply:
addressed in https://github.com/openshift/windows-machine-config-operator/commit/7e157783f4ba8bf60418d9aa73276fe4ea820dc3

Reply:
/lgtm 

looks good! :+1: